components/patient-access-rules/ui/src/main/resources/PhenoTips/AccessRightsManagement.xml

```diff
@@ -435,8 +435,10 @@
         "collaborator"  : "$escapetool.javascript($services.localization.render('phenotips.accessRightsManagement.collaboratorRole'))",
         "user"          : "$escapetool.javascript($services.localization.render('phenotips.accessRightsManagement.userType'))",
         "group"         : "$escapetool.javascript($services.localization.render('phenotips.accessRightsManagement.groupType'))",
-        "private"       : "$escapetool.javascript($services.localization.render('phenotips.permissions.visibility.private.label'))",
-        "hidden"       : "$escapetool.javascript($services.localization.render('phenotips.permissions.visibility.hidden.label'))",
+#foreach ($visibility in $services.permissions.listVisibilityOptions())
+#set ($key = "phenotips.permissions.visibility.${visibility}.label")
+        "$visibility" : "$escapetool.javascript($services.localization.render($key))",
```

Comment:
```suggestion
        "$escapetool.javascript($visibility.name)" : "$escapetool.javascript($visibility.label)",
```

The label of the visibility can be obtained directly, there's no need to manually use the l10n service.

Reply:
Great, thanks @sdumitriu